pipenv/cli.py

```diff
@@ -580,6 +583,8 @@ def ensure_virtualenv(three=None, python=None, site_packages=False):
         ensure_python(three=three, python=python)
 
         click.echo(crayons.red('Virtualenv already exists!'), err=True)
+        if not (PIPENV_YES or click.confirm('Remove existing virtualenv?', default=False)):
```

Comment:
Alright I'm going to propose an addendum here. Rather than prompting every user every time we create a new environment, let's only do this if VIRTUAL_ENV environment variable is set. That will tell us if virtualenv is already active and keep people from overwriting the potentially unrelated environment.

@erinxocon thoughts?

Reply:
I like this approach, no extra commands or env variables.  

Question: Is there any _common_ scenario where you can somehow manually activate a virtualenv without this variable being set as well?

Reply:
@nateprewitt agreed

Reply:
@erinxocon no